### PR TITLE
#391 2023

### PR DESCRIPTION
**Pull request by kirjaamo**
Copyright year updated

### CODE REVIEW

File: galeracluster/source/documentation/docker.rst
Comment (byte) [inline galeracluster/source/documentation/docker.rst:1]: What happened here @kirjaamo ?

File: galeracluster/source/documentation/quorum-reset.rst
Comment (byte) [inline galeracluster/source/documentation/quorum-reset.rst:1]: Same thing here, re: changes, between: service mysql start --wsrep-new-cluster versus sevice mysql start mysqld_bootstrap

File: galeracluster/source/training/tutorials/galera-on-aws.rst
Comment (byte) [inline galeracluster/source/training/tutorials/galera-on-aws.rst:1]: Yet another!

File: galeracluster/source/training/tutorials/starting-cluster.rst
Comment (byte) [inline galeracluster/source/training/tutorials/starting-cluster.rst:1]: Same deal here

**Review by byte — COMMENTED** [review]
copyright notices are good, but the starting stuff, what happened here

**Comment by kirjaamo** [comment]
I guess I started from a wrong branch. I'll pull the master and start over with a new branch.

**Comment by kirjaamo** [comment]
Or did you already merge to master?

**Comment by byte** [comment]
> Or did you already merge to master?

did not merge, yet

also, please check Conflicting files
galeracluster/source/_templates/layout.html

**Review by byte — APPROVED** [review]
I don't like that there are more than one change here, but it is ok, it is more than s/2022/2023 here. and soon it will be 2024

**Pull request merged**